gslib/utils/wrapped_credentials.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you please add test coverage for this file?

Reply:
Done